src/content/pages/index.md

```diff
@@ -91,6 +91,27 @@ migrationInfo:
       variant: EXTERNAL_OUTLINE
   bgColor: "#141414"
   rowReverse: false
+tacoRole:
+  title: Build uncensorable e2ee into your dapp
+  description: TACo is the only end-to-end encryption plugin that is end-to-end decentralized
+  image: /images/taco-section-image.svg
+  buttons:
+    - label: Read the docs
+      url: https://docs.threshold.network/applications/threshold-access-control
+      variant: INTERNAL_SOLID
+    - label: Quickstart
+      url: https://docs.threshold.network/app-development/threshold-access-control-tac/quickstart-testnet
+      variant: INTERNAL_OUTLINE
```

Comment:
I think those two should be external button, meaning they should have arrows next to them, because they navigate the user to the page outside the threshold.website.

```suggestion
    - label: Read the docs
      url: https://docs.threshold.network/applications/threshold-access-control
      variant: EXTERNAL_SOLID
    - label: Quickstart
      url: https://docs.threshold.network/app-development/threshold-access-control-tac/quickstart-testnet
      variant: EXTERNAL_OUTLINE
```

Current implementation:
<img width="613" alt="image" src="https://github.com/threshold-network/website/assets/40306834/9e382af2-8f7a-4588-91e0-5d57c2e7d012">

After the changes it should be
<img width="550" alt="image" src="https://github.com/threshold-network/website/assets/40306834/c122bf29-8504-49d7-ad41-138a3168aa0b">
but please double-check this.

Reply:
Makes total sense, addressed in [Refactor taco section buttons variant to external](https://github.com/threshold-network/website/pull/106/commits/97a64e21ecb9103e59f86929abe24a874eb2bc90)